test/index.test.js

```diff
@@ -60,6 +60,37 @@ describe('index.js', () => {
         expect(definition).toBeNull();
     });
 
+    it('Find by device should generate for unknown', () => {
```

Comment:
Could you move this test to `test/generateDefintion.ts` and structure it similar to the `test/modernExtend.ts`? (with `mockDevice`/`assertDefinition`)

Reply:
Added some, please check.